quickstarts/about-quickstarts.md

```diff
@@ -1,37 +1,107 @@
-# The quickstart tutorial template
 
-The quickstart or `Hello World!` tutorial is for introducing your users to your product or framework for the first time. Users can typically complete these tutorials in less than 15 minutes. It's task-based and installation-focused.
+# Quickstart Guide
 
-## About the overview section
+{Read this document before you start working on the [quickstart template](template-quickstarts.md)}
 
-Use this section to provide a short description of your product and its purpose, and to describe what the user will accomplish in this tutorial.
+## Introduction
 
-You can also state the intended audience for your tutorial and mention any concepts users should be familiar with before starting the tutorial. 
+The quickstart is intended for introducing your users to your product or framework for the first time. 
+It focuses on the **hero feature** of the product and helps your users to start using the product as quickly as possible.
 
-## About the installation section
+A good quickstart answers the following questions:
```

Comment:
This list of questions is a great addition to the quickstart guide. I can see teams using these questions to guide development of their own quickstart tutorial. Knowing where to start with a quickstart can be difficult!

I might suggest reorganizing this section a little. Some bullet points are longer than others. Maybe the top level bullet could be the question ("Where can I find the product?") and you can add the explanation as a nested bullet.

Reply:
We have logically grouped as suggested and also removed the Help based questions from the guide.